editor/find_in_files.cpp

```diff
@@ -837,6 +839,12 @@ class ConservativeGetLine {
 	Vector<char> _line_buffer;
 };
 
+void FindInFilesPanel::_update_font() {
```

Comment:
I'd suggest putting those calls directly in the `_notification` branch instead of making a dedicated method.